gpkit/nomials/nomial_math.py

```diff
@@ -591,97 +468,179 @@ def __init__(self, left, right, oper_ge=True):
                     p = p/coeff
                 elif p.cs[i] > 1:
                     raise ValueError("infeasible constraint:"
-                                     "constant term too large.")
-
-        super(Constraint, self).__init__(p)
-        self.__class__ = Constraint  # TODO should not have to do this
+                                     " constant term too large.")
+        self.p_lt, self.m_gt = p_lt, m_gt
+        self.posylt1_rep = p
+        self.substitutions = p.values
+
+    def as_posyslt1(self):
```

Comment:
method docstring? I'm conceptually confused why this requires so much code to implement -- at the `ScalarSingleEquationConstraint`, why can't this just basically return itself?